### PR TITLE
#182201663 Integrate Coveralls code coverage service

### PR DESCRIPTION
**Pull request by uwambajeddy**
####What does this PR do?
This PR is used to analyse the project's code quality. 

####Description of Task to be completed?
-Adding coveralls service to the project

####How should this be manually tested?
after cloning this PR , cd into it Run `npm install` then `npm run coverage`

####What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201663

### CODE REVIEW

**Review by alaintresor — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# bravo-bn-be
\ No newline at end of file
+[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/bravo-bn-be/badge.svg?branch=ch-code-coverage-182201663)](https://coveralls.io/github/atlp-rwanda/bravo-bn-be?branch=ch-code-coverage-182201663)
```
Comment (didasmbalanya) [inline README.md]: Wrong branch. set it to show coverage on develop

**Review by didasmbalanya — CHANGES_REQUESTED** [review]

**Review by joshwambere — COMMENTED** [review]
@uwambajeddy  can add the badges

**Comment by uwambajeddy** [comment]
> @uwambajeddy can add the badges

@joshwambere Mr Didas told me to put the coverage badge to the "develop branch" earlier. But I don't have access to the develop branch to make any changes.

**Comment by joshwambere** [comment]
> > @uwambajeddy can add the badges
> 
> @joshwambere Mr Didas told me to put the coverage badge to the "develop branch" earlier. But I don't have access to the develop branch to make any changes.

since your branch will be merged into develop, it makes sense to add into your PR, cause there is no way you'll be making changes into develop directly

**Review by joshwambere — DISMISSED** [review]

**Review by Patricknshimiyimana — DISMISSED** [review]

**Review by Placide123 — DISMISSED** [review]

**Review by mwisemarierose — DISMISSED** [review]

**Review by joshwambere — APPROVED** [review]

**Pull request merged**